example/all.yaml

```diff
@@ -127,6 +127,7 @@ linter:
     - provide_deprecation_message
     - public_member_api_docs
     - recursive_getters
+    - replace_escapes_by_raw_strings
```

Comment:
Wondering about the name...  Maybe something like `prefer_raw_strings`?  Or `unnecessary_escaped_strings`?

Reply:
I think we wanted to move away from using "prefer", etc. in the name. Perhaps `use_raw_strings`?

Reply:
Yeah.  I like `use_raw_strings` better anyway!

How does that sound @a14n ?

Reply:
SGTM. Renamed